dotnet-authserver/src/TeacherIdentity.AuthServer/IdentityLinkGenerator.cs

```diff
@@ -62,26 +60,6 @@ protected virtual string Page(string pageName, bool authenticationJourneyRequire
 
     public string TrnCallback() => Page("/SignIn/TrnCallback");
 
-    public string TrnHasTrn() => Page("/SignIn/Trn/HasTrnPage");
-
-    public string TrnOfficialName() => Page("/SignIn/Trn/OfficialName");
-
-    public string TrnPreferredName() => Page("/SignIn/Trn/PreferredName");
-
-    public string TrnDateOfBirth() => Page("/SignIn/Trn/DateOfBirthPage");
-
-    public string TrnHasNiNumber() => Page("/SignIn/Trn/HasNiNumberPage");
-
-    public string TrnNiNumber() => Page("/SignIn/Trn/NiNumberPage");
-
-    public string TrnAwardedQts() => Page("/SignIn/Trn/AwardedQtsPage");
-
-    public string TrnIttProvider() => Page("/SignIn/Trn/IttProvider");
-
-    public string TrnCheckAnswers() => Page("/SignIn/Trn/CheckAnswers");
-
-    public string TrnNoMatch() => Page("/SignIn/Trn/NoMatch");
-
```

Comment:
I think we can remove the `Update{Name|Email}` methods too.

Reply:
we can indeed!